### PR TITLE
Added a cheat code and bedlike improvements

### PR DESCRIPTION
**Pull request by pkerpedjiev**
## Description

> What was changed in this pull request?

- Added a cheat code which, if typed, toggles the header
Apr-09-2020 16-59-46

- Added `fontColor`, `fontSize` and `fillOpacity` options to the BedLikeTrack

image

- Fix track name in the config menu

image

> Why is it necessary?

Fixes #___

## Checklist

- [x] Set proper GitHub labels (e.g. v1.6+, ignore if you don't know)
- [x] Unit tests added or updated
- [x] Documentation added or updated
- [o] Example(s) added or updated
- [o] Update schema.json if there are changes to the viewconf JSON structure format
- [x] Screenshot for visual changes (e.g. new tracks or UI changes)
- [x] Updated CHANGELOG.md

### CODE REVIEW

File: app/scripts/HiGlassComponent.js [hunk 1]
```diff
@@ -831,7 +835,37 @@ class HiGlassComponent extends React.Component {
...
+  typedTextHandler(event) {
+    this.typedText = this.typedText.concat(event.key);
+
+    if (this.typedText.endsWith('hgedit')) {
+      this.toggleTypedEditable();
+      this.typedText = '';
+    }
+
+    // 1.5 seconds to type the next letter
+    const TYPED_TEXT_TIMEOUT = 1500;
```
Comment (flekschas) [inline app/scripts/HiGlassComponent.js:854]: Seems a bit long? Maybe `750` would do as well?

File: app/scripts/HiGlassComponent.js [hunk 2]
```diff
@@ -831,7 +835,37 @@ class HiGlassComponent extends React.Component {
...
   }
 
+  toggleTypedEditable() {
+    this.setState({
+      typedEditable: !this.isEditable()
+    });
+  }
+
+  /** Handle typed commands (e.g. e-d-i-t) */
+  typedTextHandler(event) {
```
Comment (flekschas) [inline app/scripts/HiGlassComponent.js:845]:
```suggestion
  typedTextHandler(event) {
    if (document.hasFocus()) return;
```

File: app/scripts/HiGlassComponent.js [hunk 3]
```diff
@@ -831,7 +835,37 @@ class HiGlassComponent extends React.Component {
...
+    this.typedText = this.typedText.concat(event.key);
```
Comment (flekschas) [inline app/scripts/HiGlassComponent.js:846]: Just out of curiosity, why not simply `this.typedText.push(event.key)`?

Comment (pkerpedjiev) [reply app/scripts/HiGlassComponent.js:851]: No good reason.

File: app/scripts/HiGlassComponent.js (hunk 1 above)
Comment (pkerpedjiev) [reply app/scripts/HiGlassComponent.js:860]: I'm a slow typer ;-) But I'll change it to 750.

**Comment by pkerpedjiev** [comment]
Thanks for the suggestions!

I used `concat` instead of `push` because it's two strings that are being joined. `document.hasFocus()` always seems to return true so I took it out.

I also added a change to allow users to pass in a custom PIXI import for using canvas.

**Pull request merged**